### PR TITLE
Refactor some notebooks

### PR DESCRIPTION
**Pull request by dopplershift**
- Separate out intro numpy/matplotlib material into their own notebook so we can assign as "homework"
- Move exercises out into their own notebooks
- Fix missing NAM data

### CODE REVIEW

File: notebooks/Time_Series/Basic Time Series Plotting Exercises.ipynb [hunk 1]
```diff
@@ -0,0 +1,203 @@
+{
```
Comment (jrleeman) [inline notebooks/Time_Series/Basic Time Series Plotting Exercises.ipynb]: Did you mean to have big commented code blocks (solutions?)

File: notebooks/Primer/Primer.ipynb [hunk 2]
```diff
@@ -2,10 +2,7 @@
  "cells": [
```
Comment (jrleeman) [inline notebooks/Primer/Primer.ipynb]: Any instructions or anything for Exercise 2? Orphan cell at the bottom of the page.

Comment (dopplershift) [inline notebooks/Primer/Primer.ipynb]: That's what I use to denote we should move over to exercise #2 in the exercise notebook, which is where the instructions live. Thoughts?

File: notebooks/Time_Series/Basic Time Series Plotting Exercises.ipynb (hunk 1 above)
Comment (dopplershift) [inline notebooks/Time_Series/Basic Time Series Plotting Exercises.ipynb]: That was intentional; the goal was to give them a skeleton of code to start with. They still need to add appropriate data-reading and imports, and then they're supposed to refactor the skeleton code to something more "loopy".

File: notebooks/Primer/Primer.ipynb (hunk 2 above)
Comment (jrleeman) [inline notebooks/Primer/Primer.ipynb]: That probably works - I think I used a smaller heading - no biggie

**Comment by jrleeman** [comment]
Looks like undefined variable issues - killing the build for now.

**Pull request merged**